### ISSUE TITLE
Conda installation in jupyter?

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#4879]
@bgruening @Delphine-L I was pointed to this tutorial recently https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/galaxy-intro-jupyter/tutorial.html which suggests you can conda install a package and have it available but I cannot reproduce this, and the two mentioned packages seem to be preinstalked in the notebook.

is there a way to conda install something and have it available? when I use `!conda info` I see no environment activated so I would suspect that nothing will be on the path correctly

cc @mbaardwijk

**Comment by Delphine-L** [comment]
@hexylena What do you get when you try to install a package ? I tried it on main with the package multivelo, and it worked fine.

**Comment by bgruening** [comment]
Yes, this should work? Which Galaxy and which Jupyter are you using?

It should also work for Rstudio.

**Comment by wm75** [comment]
> It should also work for Rstudio.

Like here, for example: https://training.galaxyproject.org/training-material/topics/evolution/tutorials/mtb_phylogeny/tutorial.html#visualize-and-manipulate-the-tree

**Comment by hexylena** [comment]
huh interesting that it works for you! 

after conda installing `python-levenshtein` it fails to become accessible;

![Screenshot_20240403-092936](https://github.com/galaxyproject/training-material/assets/458683/cc9170a7-cb40-4a0a-ac3e-d7a2a7e14a31)

**Comment by hexylena** [comment]
> Which Galaxy and which Jupyter are you using?.

eu of course, and jupylab

**Comment by wm75** [comment]
`import Levenshtein` perhaps?

**Comment by hexylena** [comment]
also fails. testing multivelo now

**Comment by hexylena** [comment]
and resetting my API key because that screenshot leaked my notebook URL which means anyone could access it 😭

**Comment by hexylena** [comment]
honestly can't tell if this is better

![Screenshot_20240403-094124](https://github.com/galaxyproject/training-material/assets/458683/5440c758-e23f-4331-a577-075cd5e19406)

**Comment by hexylena** [comment]
yeah ok multivelo imports. thanks for the tip @Delphine-L

**Comment by wm75** [comment]
Don't know what the problem with the IT context is, but in case python-levenshtein is really what you're trying to get to work: make sure you install from conda-forge (not bioconda) to get the latest version (0.25), and `import Levenshtein` (with the captial L) is definitely how you'd import it.

**Comment by hexylena** [comment]
No, it's a different package, `squidpy`, I was trying to produce a smaller/easier repro and guess I stumbled on a package that also works poorly with conda.

```
ImportError: /opt/conda/lib/python3.8/site-packages/zmq/backend/cython/../../../../.././libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /opt/conda/lib/python3.8/site-packages/pandas/_libs/window/aggregations.cpython-38-x86_64-linux-gnu.so)
```

needs to be solved some other way I guess :grimacing:

**Comment by bgruening** [comment]
fwiw, I try to fix Squidpy but got stuck at https://github.com/conda-forge/xarray-dataclasses-feedstock/pull/2

**Comment by hexylena** [comment]
Thank you for trying @bgruening :(

**Comment by hexylena** [comment]
@bgruening reports that it should be fixed https://anaconda.org/conda-forge/squidpy/files which is fantastic news.
